pixels-storage/pixels-storage-s3/src/main/java/io/pixelsdb/pixels/storage/s3/Minio.java

```diff
@@ -50,19 +51,17 @@
  * @author hank
  * @create 2022-09-04
  */
-public final class Minio extends AbstractS3
-{
+public final class Minio extends AbstractS3 {
     // private static Logger logger = LogManager.getLogger(Minio.class);
     private static final String SchemePrefix = Scheme.minio.name() + "://";
 
-    private static String minIOEndpoint = null;
-    private static String minIOAccessKey = null;
-    private static String minIOSecretKey = null;
+    private static String minIOEndpoint = System.getenv("MINIO_ENDPOINT");
+    private static String minIOAccessKey = System.getenv("MINIO_ACCESS_KEY");
```

Comment:
Don't get minio configurations from the environment variables. We can not update the env of lambda dynamically during query execution.

Reply:
Minio endpoint is set in pixels.properties and automatically passed into the cloud function workers.

Reply:
Okay, I let the input value decide minio, and give up env variables